scripts/windows/test/Setup-Env.ps1

```diff
@@ -76,7 +82,15 @@ If ($osEdition -eq "IoTUAP")    # Windows IoT Core - update iotedge
         }
         
         Write-Host "Cleanup existing containers..."
-        docker -H npipe:////./pipe/iotedge_moby_engine rm -f $(docker -H npipe:////./pipe/iotedge_moby_engine ps -aq) 2>null
+        try {
+            $residualModules = $(docker -H npipe:////./pipe/iotedge_moby_engine ps -aq)
```

Comment:
what is the type of $residualModules? array? or string? if it is array, can docker rm command handle it? tested with more than 1 module?

Reply:
_$residualModules_ is array and the docker command handles it fine.
That command for initializing _$residualModules_ fails sometimes (I am still not sure why), that's why I have the try-catch around it.
_$residualModules = $(docker -H npipe:////./pipe/iotedge_moby_engine ps -aq)_

Reply:
it will fail if docker is not installed.

Reply:
I see failures on the same machine that has docker installed, which is really strange.
See attempts 10 and 11 here: https://msazure.visualstudio.com/One/_releaseProgress?releaseId=971990&environmentId=2399693&_a=release-environment-logs